### DISCUSSION TITLE
New user question about server/distro selection

### DISCUSSION THREAD

**Discussion by Gene-W** [nextcloud/all-in-one#3289] [Questions] [answered]
Hello, I'm really new at attempting to self-host so this question will probably reflect that!

At the very beginning of the AIO instructions, I see this:
`The following instructions are meant for installations without a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else) already being in place. If you want to run AIO behind a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else), see the [reverse proxy documentation](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md). Also, the instructions below are especially meant for Linux.`

My question is, are the instructions following this statement meant for installation on a distro that's not a server? I've been considering using either Fedora or Ubuntu server, but if it's easier to use a non-server distro I can do that.

Thanks for any direction on server/distro selection!

**Comment by Zoey2936** [comment, marked as answer]
ubuntu and fedora should both work

**Reply by szaimen** [reply]
Yes, I would probably recommens Ubuntu Server 22.04 LTS

**Reply by Gene-W** [reply]
Thank you @Zoey2936 and @szaimen - curious about Ubuntu, have you found that it may work more seamlessly with AIO, or is it recommended for other reasons?

**Reply by szaimen** [reply]
> curious about Ubuntu, have you found that it may work more seamlessly with AIO

Ubuntu works more seamless with Docker

**Reply by Gene-W** [reply]
Excellent, I will go with that and see what I can accomplish. Thank you!

**Reply by Zoey2936** [reply]
I've never used RPM based OS, so no Idea, I always use Debian, which works perfect for me. Ubuntu is based on Debian. But since AIO is based on Docker, it should work on all OS, not sure if Docker works better on some OS. Only OS with SE-Linux need to do this https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled.
